src/Native/Windows/clrcompression/zlib-intel/adler32.c

```diff
@@ -0,0 +1,195 @@
+/* adler32.c -- compute the Adler-32 checksum of a data stream
+ * Copyright (C) 1995-2011 Mark Adler
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+/* @(#) $Id$ */
```

Comment:
[Other source control systems](http://svnbook.red-bean.com/en/1.4/svn.advanced.props.special.keywords.html) expand that on checkout to something meaningful (e.g. the filename, last modified date, etc).